quickstart.sql

```diff
@@ -0,0 +1,93 @@
+-- Quickstart PL/SQL template to launch an ECS fargate service.
+-- Cofigure your project updating the first set of variables.
+
+do $$
+<<quickstart>>
+  declare
+    project_name text := 'quickstart';
```

Comment:
can this be a parameter that we pass in?

Reply:
I was not sure, but yes, implemented!